apps/meteor/client/views/marketplace/AppsProvider.tsx

```diff
@@ -241,19 +271,38 @@ const AppsProvider: FC = ({ children }) => {
 				apps: installedAppsData,
 			});
 		}
+
+		if (!privateAppsError) {
+			if (privateApps.length) {
```

Comment:
Using typescripts boolean casting here may be a good idea, would make the code more readable. e.g.: if (Boolean(privateApps.length))

Reply:
hmmmm, ok... I will put a `privateApps.length > 0`, sounds better indeed